spec/lang/well-formed.md

```diff
@@ -31,17 +31,19 @@ impl IntType {
 impl PointeeInfo {
     fn check_wf<T: Target>(self) -> Result<()> {
         // We do *not* require that size is a multiple of align!
-        ensure_wf(T::valid_size(self.size), "Layout: size not valid")
+        ensure_wf(T::valid_size(self.size), "PointeeInfo: size not valid")
     }
 }
 
 impl PtrType {
     fn check_wf<T: Target>(self) -> Result<()> {
         match self {
-            PtrType::Ref { pointee, mutbl: _ } | PtrType::Box { pointee } => {
+            PtrType::Ref { pointee, meta_kind, .. } | PtrType::Box { pointee, meta_kind } => {
+                // TODO: ensure pointee size and align fit with metadata kind.
```

Comment:
```suggestion
                // TODO(UnsizedTypes): ensure pointee size and align fit with metadata kind.
```
Please mark all TODOs like that. Then we can easily grep later in the semester to see if we forgot any of them.